resources/aux/keycloak-template

```diff
@@ -0,0 +1,20 @@
+version: '3.1'
```

Comment:
Not sure why the template files aren't named with the `.yml` file extension.

Reply:
yes, better to keep it for IDE readability

Reply:
done